Makefile

```diff
@@ -1,15 +1,34 @@
 all: examples
```

Comment:
There is much better way! 

Just `go get github.com/bwplotka/bingo@v0.2.1` (or bingo get)

then run `bingo get` and here you can put `include .bingo/Variables.mk`. Then the FUN begins when you can just use variables, no need for manual copy of  all those: 
```
.bingo/jb:
	go build -modfile .bingo/jb.mod -o .bingo/jb github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb

.bingo/jsonnet:
	go build -modfile .bingo/jsonnet.mod -o .bingo/jsonnet github.com/google/go-jsonnet/cmd/jsonnet

.bingo/jsonnetfmt:
	go build -modfile .bingo/jsonnetfmt.mod -o .bingo/jsonnetfmt github.com/google/go-jsonnet/cmd/jsonnetfmt

.bingo/gojsontoyaml:
	go build -modfile .bingo/gojsontoyaml.mod -o .bingo/gojsontoyaml github.com/brancz/gojsontoyaml
```

:hugs: 

Similar to: https://gitlab.cee.redhat.com/observatorium/configuration/merge_requests/87